examples/login/login.py

```diff
@@ -1,61 +1,58 @@
+"""This example demonstrates how to log in to a website using AgentQL."""
+
 import time
-import agentql
 
-URL = "https://www.target.com"
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+URL = "https://www.yelp.com/"
```

Comment:
```suggestion
URL = "https://target.com/"
```

Reply:
I've actually changed the example website from `target` to `yelp` because there are too many distracting elements on `target` website (such as promotion to skip before clicking the login button). I feel like for the purpose of this example (demonstrating how to login), `yelp` is a better choice. Let me know your thoughts!

Btw, as I prepare these examples, I feel like probably we want to include more examples showing different things, such as formatting the response with query contexts, showing how to handle popup window, how to scroll the page up and down, etc. Once the current examples are fixed and included in documentation, I will compile a list and share those with you.